docs/modules/integrate/pages/jdbc-connector.adoc

```diff
@@ -45,11 +45,11 @@ p.readFrom(Sources.jdbc(
 )).writeTo(Sinks.logger());
 ```
 
-You can also use a configured xref:external-data-stores:external-data-stores.adoc#defining-external-data-stores[external data store] as a JDBC Source:
+You can also use a configured xref:external-data-stores:external-data-stores.adoc#defining-external-data-stores[data connection] as a JDBC Source:
```

Comment:
This page doesn't exist...looks like data-connections-configuration.adoc has the alias and the link should still work

Reply:
The link does still work, but the updated link text wasn't showing up in the build for some reason. I've just updated all these xrefs (and any child section refs) to point to the new file.